modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/WikiPageResourceImpl.java

```diff
@@ -362,6 +363,20 @@ protected String getPermissionCheckerResourceName(Object id) {
 			contextAcceptLanguage.getPreferredLocale());
 	}
 
+	private String _mapEncodingFormat(String encodingFormat) {
```

Comment:
I like this name, map is very well used. But I guess it's going tok be renamed to `to`